composer.json

```diff
@@ -16,7 +16,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^6.0|^7.0|^8.0",
```

Comment:
You also need to add ^5.0 here.

Reply:
I'm checking locally and seems that Phpunit 6 works also for orchestra/testbench 3.4, which is required to test on Laravel 5.4

Reply:
You're right as the base is PHP 5.6 not 7